### DISCUSSION TITLE
is there any way to customize no.of recent transactions to show in account modal

### DISCUSSION THREAD

**Discussion by ethdev279** [rainbow-me/rainbowkit#717] [Ideas]
is there any way to customize no.of recent transactions to show in account modal? right now it is showing only  3 most recent transactions. if we can customize no.of transactions, that will be great(with scrolling for ux)

![Screenshot (166)](https://user-images.githubusercontent.com/45661693/188298474-62736578-b505-47a3-9ade-40e5267e376d.png)

**Comment by johnsmithberlin** [comment]
Bro share with me your transaction history code
